addons/MH60S/config/Pylons.hpp

```diff
@@ -6,7 +6,7 @@ class TransportPylonsComponent {
             attachment = "VTX_4Rnd_ACE_Hellfire_AGM114K";
             priority = 5;
             hardpoints[] = {
-              "DAR", "DAGR", "B_SHIEKER", "20MM_TWIN_CANNON",
```

Comment:
Forgotten Comma

Reply:
this file doesn't looks like it should have any changes from main.